pkg/controller/deployment/util/deployment_util.go

```diff
@@ -816,9 +816,10 @@ func IsRollingUpdate(deployment *extensions.Deployment) bool {
 }
 
 // DeploymentComplete considers a deployment to be complete once its desired replicas equals its
```

Comment:
We'll need to modify docs https://kubernetes.io/docs/user-guide/deployments/#complete-deployment